src/nools/target-emitter.js

```diff
@@ -21,6 +21,39 @@ function targetEmitter(targets, c, Utils, Target, emit) {
   }
 }
 
+function determineDate(targetConfig, Utils, c, r) {
+  if (typeof targetConfig.date === 'function') {
+    return targetConfig.date(c, r);
+  }
+  
+  if (targetConfig.date === undefined || targetConfig.date === 'now') {
+    return Utils.now().getTime();
+  }
+
+  if (targetConfig.date === 'reported') {
+    return r ? r.reported_date : c.contact.reported_date;
+  }
+  
+  throw new Error('Unrecognised value for target.date: ' + targetConfig.date);
+}
+
+function determineInstanceIds(targetConfig, c, r) {
+  var instanceIds;
```

Comment:
Use `let`.

Reply:
The code that runs inside nools still uses eslint rules which enforce ES5 :(

Reply:
Of course  :(